pkg/operator/ceph/cluster/osd/config_test.go

```diff
@@ -33,3 +33,39 @@ func TestOsdOnSDNFlag(t *testing.T) {
 	args = osdOnSDNFlag(network)
 	assert.Empty(t, args)
 }
+
+func TestEncryptionKeyPath(t *testing.T) {
+	tests := []struct {
+		name string
+		want string
+	}{
+		{"default", "/etc/ceph/luks_key"},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := encryptionKeyPath(); got != tt.want {
+				t.Errorf("encryptionKeyPath() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func TestGenerateOSDEncryptionSecretName(t *testing.T) {
+	type args struct {
+		osdID int
+	}
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		{"default", args{3}, "rook-ceph-osd-key-3"},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
```

Comment:
Is it really worth using this test pattern everywhere? It seems like a complex validation just to check one string. For example, this whole test could have been a single line of code:
```
assert.Equal(t, "rook-ceph-osd-key-3", generateOSDEncryptionSecretName(0))
```

Reply:
Yeah, since I realized vscode was generating this for me I got used to it but it does not suit well when there is a single validation.